src/PrestaShopBundle/Translation/Provider/AbstractProvider.php

```diff
@@ -208,7 +208,7 @@ public function getDatabaseCatalogue($theme = null)
         $databaseCatalogue = new MessageCatalogue($this->locale);
 
         foreach ($this->getTranslationDomains() as $translationDomain) {
-            $domainCatalogue = $this->getDatabaseLoader()->load(null, $this->locale, $translationDomain, $theme);
+            $domainCatalogue = $this->getDatabaseLoader()->load(null, $this->locale, $translationDomain);
```

Comment:
⚠️ DatabaseTranslationLoader **has** a `$theme` argument
```
public function load($resource, $locale, $domain = 'messages', $theme = null)
```

I think phpstan complains because the interface does not have this argument. Once again, you and PHPStan have found a usecase where implementation and interface do not have the same signature 😭 

We cannot remove `$theme` I think it would introduce bugs.

Reply:
I fixed the `$theme` and the return value for `getDatabaseLoader`.